### PR TITLE
remove job entries when drop space && check space exist when check jo…

### PR DESCRIPTION
**Pull request by pengweisong**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [ ] enhancement

#### Issue(s) number: 
https://github.com/vesoft-inc/nebula-ent/issues/1643

#### Description:
When drop the space, the job entries have not been dropped. Then we create backup, it may have some job entries left which are rebuilding jobs. In this case, there is no rebuilding job really, but the backup creation is blocked.

## How do you solve it?
1. drop the job entries when drop spaces.
2. skip the spaces not exist when check rebuilding jobs.

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by critical27 — APPROVED** [review]

**Review by SuperYoko — APPROVED** [review]

File: src/meta/processors/job/JobManager.cpp [hunk 1]
```diff
@@ -66,6 +66,16 @@ JobManager::~JobManager() {
...
 }
 
+bool JobManager::spaceExist(GraphSpaceID spaceId) {
+  auto spaceKey = MetaKeyUtils::spaceKey(spaceId);
+  std::string val;
+  auto retCode = kvStore_->get(kDefaultSpaceId, kDefaultPartId, spaceKey, &val);
+  if (retCode == nebula::cpp2::ErrorCode::E_KEY_NOT_FOUND) {
+    return false;
+  }
+  return true;
```
Comment (SuperYoko) [inline src/meta/processors/job/JobManager.cpp:76]: This judge looks dangerous, but since true do nothing, I think it's okay.

Comment (pengweisong) [reply src/meta/processors/job/JobManager.cpp:76]: OK, I'll make it explicit.

Comment (SuperYoko) [reply src/meta/processors/job/JobManager.cpp:76]: > 

I'm okay with current code. If you modify, then better is better ovo

Comment (pengweisong) [reply src/meta/processors/job/JobManager.cpp:75]: I've changed it, but forget to push it before it was merged..

Comment (SuperYoko) [reply src/meta/processors/job/JobManager.cpp:75]: > 

Don't worry, I don't think it must be changed.

**Pull request merged**